index.html

```diff
@@ -9,184 +9,193 @@
+            <button type="button">Get My resume</button>
+
+            <div class="languages">Languages <i class="fa-solid fa-angle-down"></i>
+              <div  class="languages"><img src="images/javascript.png" alt="Javascript">
+              <p>Javascript</p></div>
+              <div  class="languages"><img src="images/html.png" alt="HTML">
+               <p>HTML5</p></div>
+              <div  class="languages"><img src="images/css.png" alt="CSS">
+                 <p>CSS3</p></div>
+              </div>
+
+
+
+        
+        <div class="languages">Frameworks<i class="fa-solid fa-angle-down"></i>
+            
         </div>
-        <section>
-            <ul class="lists">
-                <li class="layout collapse">
-                    <div class="textlayout">Languages</div><img class="vector" src="vector1.png" alt="vector">
-                </li>
-                <li class="layout frame"><img class="pad" src="javascript.png" alt="js">
-                    <div class="textLi">Javascript</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="html.png" alt="html">
-                    <div class="textLi">HTML</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="css.png" alt="css">
-                    <div class="textLi">CSS</div>
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Frameworks</div><img class="vector1" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Skills</div><img class="vector2" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
 
-        </section>
+
+        <div class="skills">Skills<i class="fa-solid fa-angle-down"></i>
+            
+        </div>
+
+    </section>
+
 
 
+    <section class="contact-me">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I'd love to help with it </p>
 
+        <form action="https://formspree.io/f/xeqngele" method="post">
+            <ul>
+                <li>
+                    <input type="text" name="username" placeholder="Enter your username" maxlength="30" required>
+                </li>
+                <li>
+                    <input type="email" name="email" placeholder="Enter your email address" required>
+                </li>
+                <li>
+                    <textarea placeholder="Write a message here" maxlength="500" required></textarea>
+                </li>
+            </ul>
+            <button type="submit" id="submit-button">Get in Touch</button>
```

Comment:
Good job on implementing a button interaction for "Hover", but it is not of the correct colour.

As per the project requirements, button interactions should be implemented.

![image](https://user-images.githubusercontent.com/81232315/152343990-ef759087-6283-4330-a20c-21f3d6268181.png)

The button interactions should be styled as per the Figma template:

![image](https://user-images.githubusercontent.com/81232315/154232696-aba03b4e-dbe6-460e-addb-c2668b109149.png)

- [ ] The colour for "Hover" should be `#6070FF`.

![image](https://user-images.githubusercontent.com/81232315/154232811-03ec2214-6974-41f3-b61e-71e28c521adc.png)

- [ ] There should also be a button interaction for "Pressed".

**_Hint:_** you can use the [`:active`](https://developer.mozilla.org/en-US/docs/Web/CSS/:active) CSS pseudo-class.